linera-storage/build.rs

```diff
@@ -5,7 +5,7 @@ fn main() {
     cfg_aliases::cfg_aliases! {
         with_testing: { any(test, feature = "test") },
         with_metrics: { all(not(target_arch = "wasm32"), feature = "metrics") },
-        with_wasmer: { all(not(target_arch = "wasm32"), feature = "wasmer") },
+        with_wasmer: { all(any(feature = "web", not(target_arch = "wasm32")), feature = "wasmer") },
```

Comment:
So without the `web` feature, Wasmer is always disabled?

Reply:
Wasmer is enabled iff the `wasmer` feature is enabled and:
- we aren't targeting Wasm, or
- the `web` feature is enabled